docs/source/installation.md

```diff
@@ -16,183 +16,182 @@ limitations under the License.
 
-Do note that you have to keep that `transformers` folder around and not delete it to continue using the  `transformers` library.
+<Tip warning={true}>
+You must keep the `transformers` folder if you want to keep using the library.
+</Tip>
 
-Now, let's get to the real benefit of this installation approach. Say, you saw some new feature has been just committed into `master`. If you have already performed all the steps above, to update your transformers to include all the latest commits, all you need to do is to `cd` into that cloned repository folder and update the clone to the latest version:
+Now you can easily update your clone to the latest version of `transformers` with the following commands:
 
-```
+```bash
 cd ~/transformers/
 git pull
 ```
 
-There is nothing else to do. Your python environment will find the bleeding edge version of `transformers` on the next run.
+Your Python environment will find the `master` version of `transformers` on the next run.
 
+## Installation with conda
 
-## With conda
+Install from the conda channel `huggingface`:
 
-Since Transformers version v4.0.0, we now have a conda channel: `huggingface`.
-
-🤗 Transformers can be installed using conda as follows:
-
-```
+```bash
 conda install -c huggingface transformers
 ```
 
-Follow the installation pages of TensorFlow, PyTorch or Flax to see how to install them with conda.
-
-## Caching models
-
-This library provides pretrained models that will be downloaded and cached locally. Unless you specify a location with
-`cache_dir=...` when you use methods like `from_pretrained`, these models will automatically be downloaded in the
-folder given by the shell environment variable ``TRANSFORMERS_CACHE``. The default value for it will be the Hugging
-Face cache home followed by ``/transformers/``. This is (by order of priority):
-
-  * shell environment variable ``HF_HOME``
-  * shell environment variable ``XDG_CACHE_HOME`` + ``/huggingface/``
-  * default: ``~/.cache/huggingface/``
-
-So if you don't have any specific environment variable set, the cache directory will be at
-``~/.cache/huggingface/transformers/``.
+# Cache setup
 
-**Note:** If you have set a shell environment variable for one of the predecessors of this library
-(``PYTORCH_TRANSFORMERS_CACHE`` or ``PYTORCH_PRETRAINED_BERT_CACHE``), those will be used if there is no shell
-environment variable for ``TRANSFORMERS_CACHE``.
+Pretrained models are downloaded and locally cached at: `~/.cache/huggingface/transformers/`. This is the default directory given by the shell environment variable `TRANSFORMERS_CACHE`. You can change the shell environment variables shown below - in order of priority - to specify a different cache directory:
 
-### Offline mode
+1. Shell environment variable: `HF_HOME`.
+2. Shell environment variable: `XDG_CACHE_HOME` + `/huggingface/`.
+3. Shell environment variable (default): `TRANSFORMERS_CACHE`.
```

Comment:
This is wrong, `TRANSFORMERS_CACHE` as priority, so this list should be:

1. Shell environment variable `TRANSFORMERS_CACHE`
2. Shell environment variable `HF_HOME` + `/transformers/`
3. Shell environment variable: `XDG_CACHE_HOME` + `/huggingface/transformers`
4. Default `~/.cache/huggingface/transformers/`

Reply:
> these models will automatically be downloaded in the folder given by the shell environment variable TRANSFORMERS_CACHE. The default value for it will be the Hugging Face cache home followed by /transformers/.

Ah ok! This part here from the current docs makes it sound like the default value of `TRANSFORMERS_CACHE` is `~/.cache/huggingface/transformers/`. So are **1** and **4** the same then (sorry if I'm missing something super obvious here 😅 )?